src/components/layout/CyVerseAppBar.js

```diff
@@ -372,15 +372,12 @@ function CyverseAppBar(props) {
         }
     }, [userProfile]);
 
-    useBootStrap(
-        bootstrapQueryEnabled,
-        (respData) => {
-            setPreferences(respData.preferences);
-            const workspace = respData["apps_info"].workspace;
-            setNewUser(workspace["new_workspace"] || false);
-        },
-        setBootstrapError
-    );
+    useBootStrap(bootstrapQueryEnabled, setBootstrapInfo, setBootstrapError);
+
+    useEffect(() => {
+        const workspace = bootstrapInfo?.apps_info.workspace;
+        setNewUser(workspace?.new_workspace);
+    }, [bootstrapInfo]);
```

Comment:
I added this `useEffect` for `bootstrapInfo` after rebasing on master, and it appears to work in Storybook, but I haven't tested this in dev mode yet.